README.md

```diff
@@ -32,7 +32,11 @@ at their own discretion.
     help projects adopt and iterate on policies and processes that achieve
     these goals and eventually allow them to graduate the incubation phase.
 
-      * Mikeal Rogers (@mikeal)
+      * Mikeal Rogers (@mikeal) Currently assigned to Express.
+      * James Snell (@jasnell) Currently assigned to Express.
+      * Rod Vagg (@rvagg) Currently assigned to libuv.
+      * Alexis Campailla (@orangemocha) Currently assigned to libuv.
+
```

Comment:
Sure, thanks :) I remembered that we avoided it on the README of nodejs/node, but I don't remember the details of why we did that.
In any case, it's not a big deal and we can fix it later.

And I assume you didn't get a notification for https://github.com/orangemocha/test/commit/9b51d8aa674de6bca3e2d785cd6099fbaaa2b6ce :)